src/movie/movie.resolver.ts

```diff
@@ -1,16 +1,25 @@
-import { Resolver, Query, Args } from '@nestjs/graphql'
+import { Args, Parent, ResolveField, Resolver, Query } from '@nestjs/graphql'
 import { MovieService } from './movie.service'
+import { Movie } from 'src/graphql'
 
 @Resolver('Movie')
 export class MovieResolver {
   constructor(private readonly movieService: MovieService) {}
 
   @Query()
-  async movie(@Args('imdbId') imdbId: string) {
+  async movie(@Args('imdbId') imdbId: string): Promise<Movie> {
     if (imdbId) {
-      return this.movieService.getMovieByImdbId(imdbId)
+      return await this.movieService.getMovieByImdbId(imdbId)
```

Comment:
It is never necessary to "return await" as I think ESLint should also signify

Reply:
Fair enough (Y)